src/linear_solvers/mkl_direct.jl

```diff
@@ -0,0 +1,61 @@
+struct MKLDirectSolver <: LinearSolver end
```

Comment:
Needs copyright header

Reply:
added;

out of curiosity: what is the rationale of adding those everywhere?

Reply:
To avoid people copy-pasting parts of the wrappers into other projects and ignoring the license implications :smile:

Reply:
but to fool copilot we'd need to scatter this notice in comments throughout the whole codebase :rofl: